src/binaryen-c.h

```diff
@@ -122,6 +122,10 @@ BinaryenExpressionId BinaryenSIMDReplaceId(void);
 BinaryenExpressionId BinaryenSIMDShuffleId(void);
 BinaryenExpressionId BinaryenSIMDBitselectId(void);
 BinaryenExpressionId BinaryenSIMDShiftId(void);
+BinaryenExpressionId BinaryenMemoryCopyId(void);
+BinaryenExpressionId BinaryenDataDropId(void);
+BinaryenExpressionId BinaryenMemoryCopyId(void);
```

Comment:
`BinaryenMemoryCopyId` appears to be duplicated?

Reply:
It seems this should be `BinaryenMemoryInitId`